qiskit_experiments/library/calibration/analysis/fine_amplitude_analysis.py

```diff
@@ -1,158 +1,159 @@
-# This code is part of Qiskit.

```

Comment:
I'm not sure why GitHub is showing this entire file as changed.

Reply:
My guess is it's different newline characters (windows vs unix)